hasura.planx.uk/migrations/1724408755595_populate_team_settings_submissions_email/up.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE team_settings
+SET submission_email = teams.submission_email
+FROM teams
+WHERE team_settings.team_id = teams.id;
```

Comment:
This all looks good! 

nit: I would prefer to see this `UPDATE...` statement squashed together into the same migration as the new column - then it will make more sense that this one doesn't have a corresponding `down.sql` !